src/reducers/users.spec.tsx

```diff
@@ -0,0 +1,62 @@
+import reducer, {
+  actions,
+  createInternalUser,
+  getCurrentUser,
+  initialState,
+} from './users';
+import { fakeUser } from '../test-helpers';
+
+describe(__filename, () => {
+  describe('reducer', () => {
+    it('initializes the state', () => {
+      const state = reducer(undefined, { type: 'SOME_ACTION' } as any);
+
+      expect(state).toEqual(initialState);
```

Comment:
fwiw, I don't think these types of tests are very helpful. We have a strongly recognized pattern for creating reducers and they always have a default case that returns the state. I'm. pretty sure we'd catch a mistake there in code review.